### PR TITLE
feat: Perk Store Add palceholder when no products displayed - MEED-1259

### PR DESCRIPTION
**Pull request by ArfaouiMohamedAmine**
this change is going to implement a placeholder when no data displayed

### CODE REVIEW

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    clickCondition: {
+      type: Boolean,
+      default: false
+    },
+  },
+  created() {
+    document.addEventListener('perk-store-products-no-results', this.noResultEvent);
+  },
+  beforeDestroy() {
+    document.removeEventListener('perk-store-products-no-results', this.noResultEvent);
```
Comment (boubaker) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue]:
```suggestion
    document.addEventListener(this.emptyProductsActionName, this.noResultEvent);
  },
  beforeDestroy() {
    document.removeEventListener(this.emptyProductsActionName, this.noResultEvent);
```

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      <v-icon
+        size="120"
+        color="primary">
+        {{ icon }}
+      </v-icon>
+      <div class="d-flex flex-column text-center text-sub-title">
+        <span v-if="info" class="text-header-title text-light-color py-5">{{ info }}</span>
+        <span
+          v-if="infoMessage"
+          v-html="emptyPerkstoreSummaryText"
```
Comment (MayTekayaa) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]: Use v-sanitized-html instead of v-html

Comment (boubaker) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]:
```suggestion
          v-sanitized-html="emptyPerkstoreSummaryText"
```

Comment (ArfaouiMohamedAmine) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]: @boubaker  v-sanitized-html has blocked the propagation of the event, v-html has worked fine !

Comment (ArfaouiMohamedAmine) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]: v-sanitized-html has blocked the propagation of the event, v-html has worked fine ! @MayTekayaa

Comment (boubaker) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]: ok, got it, knowing that this is just an I18N and not user provided message, so it's minor. Thanks @ArfaouiMohamedAmine .

Comment (boubaker) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue:29]: ok, got it, knowing that this is just an I18N and not user provided message, so it's minor. Thanks @ArfaouiMohamedAmine .

**Review by boubaker — APPROVED** [review]

**Pull request merged**